src/lib/components/TextAppearanceSelector.svelte

```diff
@@ -22,14 +22,14 @@ The navbar component.
         if (countThemesNormal && countThemesSepia && countThemesDark) {
             return true;
         } else if (countThemesNormal && countThemesSepia) {
-        /** 2 theme are enabled*/
-        /** If Normal and Sepia are enabled*/
+            /** 2 theme are enabled*/
```

Comment:
This is too much code.  This could simply be:

```js
    let countThemes = $globalConfig.themes.filter((x) => x.enabled).length;

    let showTextAppearence =
        $globalConfig.mainFeatures['text-font-size-slider'] ||
        $globalConfig.mainFeatures['text-line-height-slider'] ||
        countThemes > 1;
```

I don't think you need reactivity (using $: before variable) with the readable store since it isn't changing. I am checking on that.